packages/admin/admin/src/form/Field.tsx

```diff
@@ -76,6 +76,7 @@ export function Field<FieldValue = any, FieldElement extends HTMLElement = HTMLE
                 disabled={disabled}
                 error={shouldShowError(meta) && (meta.error || meta.submitError)}
                 warning={shouldShowWarning(meta) && meta.data?.warning}
+                helperText={meta.data?.helperText}
```

Comment:
helperText will be never be in meta, but should be a root prop of the field. Also, the prop is missing in `FieldProps`